living_documentation_generator/generator.py

```diff
@@ -92,22 +78,32 @@ def github_instance(self) -> Github:
     @property
     def repositories(self) -> list[ConfigRepository]:
         """Getter of the list of config repository objects to fetch from."""
-        return self.__repositories
+        return self.__action_inputs.repositories
 
     @property
     def project_state_mining_enabled(self) -> bool:
         """Getter of the project state mining switch."""
-        return self.__project_state_mining_enabled
+        return self.__action_inputs.is_project_state_mining_enabled
 
     @property
     def structured_output(self) -> bool:
         """Getter of the structured output switch."""
-        return self.__structured_output
+        return self.__action_inputs.structured_output
 
     @property
     def output_path(self) -> str:
         """Getter of the output directory."""
-        return self.__output_path
+        return self.__action_inputs.output_directory
+
+    @property
+    def github_projects_instance(self) -> GithubProjects:
+        """Getter of the GitHub projects instance."""
+        return self.__github_projects_instance
+
+    @property
+    def safe_call(self) -> Callable:
```

Comment:
Why we need this when it is used only inside the class methods.

Reply:
We had it also for github_instance, which confused me. I will get rid off getter for github_instance, github_projects_instance and safe_call.

Reply:
Fixed in commit: [0d7be27](https://github.com/AbsaOSS/living-doc-generator/pull/40/commits/0d7be2743ed42800deb71da0793a64636f075e85).